src/gatsby/gatsby-browser.js

```diff
@@ -1,11 +1,28 @@
+import { newTracker, enableActivityTracking, trackPageView } from '@snowplow/browser-tracker';
 import "@bcgov/bootstrap-theme/dist/css/bootstrap-theme.min.css"
 import "@bcgov/bc-sans/css/BC_Sans.css"
 import "jquery/dist/jquery.slim"
 import "bootstrap/dist/js/bootstrap.bundle"
 import "@fortawesome/fontawesome-free/css/all.min.css"
 import "./src/styles/style.scss"
 
+// Snowplow tracking
```

Comment:
Is it possible to use the snowplow instance that's already loaded instead of loading another one?
e.g. https://bcparks.ca/_scripts/snowplow.js is already loaded in `bcparks.ca/src/gatsby/gatsby-ssr.js`
```
const HeadComponents = [
    process.env.GATSBY_ENABLE_SNOWPLOW === "true" &&
    <script src="/_scripts/snowplow.js" key="snowplow-js" />,
    ...
];
```

Reply:
For workbc it looks like we used the windows object for custom snowplow tracking.  
https://github.com/search?q=repo%3Abcgov%2Fworkbc-jb%20snowplow&type=code
workbc was typescript, so we had to cast `window as any`, but that won't be needed in js.

Reply:
I think calls to `refreshLinkClickTracking` like on workBC will probably also be needed in most cases.
https://docs.snowplow.io/docs/collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/tracking-events/link-click/

Reply:
Seems these two are a bit different. The `gatsby-ssr.js` is used during the server-side rendering process. This script tag includes the Snowplow JavaScript file (snowplow.js) in the pages. This is necessary for the Snowplow tracking to work. The `gatsby-browser.js` is used for customizing and extending default settings affecting the browser. This is where you initialize the Snowplow tracker and set up event tracking. This code runs in the browser after the page has loaded. But I'm not 100% sure tbh, I couldn't find many examples of Gatsby and Snowplow 🥲